calibration/include/calibration/BoloProperties.h

```diff
@@ -23,6 +23,7 @@ class BolometerProperties : public G3FrameObject
 	double y_offset; /* and el */
 
 	double band;		/* Standard frequency units */
+	double center_frequency;
```

Comment:
If the SLIM detectors have more properties here (the 3G ones do have this property, but from an implementation point of view), it might make sense to subclass this rather than modifying the class. That also provides an opportunity to rename the subclass if people object to "bolometer" without touching any of the 3G codebase.

Reply:
It sounds like this is the only additional property, but maybe @adamanderson can confirm.

Reply:
I was thinking of this basically as an excuse for future-proofing, if you wanted one.

Reply:
I think making a subclass just to add a single property adds too much duplicate boilerplate code.  It's probably easier to (eventually) just rename Bolometer->Detector in a backward-compatible way and be done with it.

Reply:
I will point out that the SLIM detectors *are* bolometers, just not TESes.

Reply:
Subclassing or hardcoding here is fine with me. There will be a few additional properties added, following discussions with Jessica, Kirit, and Pete.

Reply:
What kinds of properties? Center frequency applies to TESes as well, though we conflate it with nominal frequency on SPT. If they are really different/extra things, I think a subclass would make a lot of sense.